text/0098-promotion-phase.md

```diff
@@ -0,0 +1,47 @@
+# Promotion Phase for GitOps use cases
+
+Add a new lifecycle phase after post deployment dedicated to GitOps promotion tasks.
+
+**State: DRAFTING**
+
+## Motivation
+
+Since one of the main use cases for Keptn is enhancement of GitOps workflows, we should have bigger focus on that
```

Comment:
I'd suggest dropping the GitOps wording in this section. The promotion task could potentially promote by applying the manifests directly into another cluster 😅

Reply:
fixed